### PR TITLE
Bugfix/#2998 add router links to sidebar

### PR DESCRIPTION
**Pull request by YulianDev**
Add router links and removed duplicate paths.

### CODE REVIEW

**Review by rostikdoroshenko — APPROVED** [review]

**Review by YuriShevchuk — APPROVED** [review]

**Review by dyadyushko — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-admin-sidebar/ubs-admin-sidebar.component.ts [hunk 1]
```diff
@@ -18,11 +18,13 @@ export class UbsAdminSidebarComponent extends UbsBaseSidebarComponent implements
     },
     {
       link: 'assets/img/sidebarIcons/shopping-cart_icon.svg',
-      name: 'Замовлення'
+      name: 'Замовлення',
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-admin-sidebar/ubs-admin-sidebar.component.ts:21]: this should probably be translated as well

Comment (JimHolman1199) [reply src/app/ubs-admin/components/ubs-admin-sidebar/ubs-admin-sidebar.component.ts:21]: you can use key for JSON file, and translate pipe in template

Comment (JimHolman1199) [reply src/app/ubs-admin/components/ubs-admin-sidebar/ubs-admin-sidebar.component.ts:21]: *applicable for all sidebar

**Review by tkhristina — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.scss [hunk 2]
```diff
@@ -17,18 +17,78 @@
...
+      padding: 0;
+    }
+
+    img {
+      padding: 0;
+    }
+  }
+
+  .add-shadow {
+    background-color: #fff;
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-header/ubs-header.component.scss:72]: we are using vars for colors

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.ts [hunk 3]
```diff
@@ -1,8 +1,42 @@
...
   templateUrl: './ubs-header.component.html',
   styleUrls: ['./ubs-header.component.scss']
 })
-export class UbsHeaderComponent {}
+export class UbsHeaderComponent implements OnInit {
+  public langDropdownVisible = false;
+  public arrayLang: Array<LanguageModel> = [
+    { lang: 'Ua', langName: 'ukrainian' },
+    { lang: 'En', langName: 'english' }
+  ];
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-header/ubs-header.component.ts:16]: it should be on global level, find one or create

**Pull request merged**